client_test.go

```diff
@@ -3656,6 +3656,17 @@ func Test_Client_Maintenance(t *testing.T) {
 	})
 }
 
+type runOnceScheduler struct {
```

Comment:
Maybe call this a "schedule" instead of a "scheduler" given the existing naming of types like `PeriodicSchedule` and `DefaultReindexerSchedule`.